src/store/chain/common/wallet/wallet.js

```diff
@@ -0,0 +1,235 @@
+import Vue from 'vue'
+import { DirectSecp256k1HdWallet } from '@cosmjs/proto-signing'
+import {
+	assertIsBroadcastTxSuccess,
+	SigningStargateClient
+} from '@cosmjs/stargate'
+import { stringToPath } from '@cosmjs/crypto'
+import CryptoJS from 'crypto-js'
+
+export default {
+	namespaced: true,
+	state() {
+		return {
+			wallets: JSON.parse(window.localStorage.getItem('wallets')),
+			activeWallet: null,
+			activeClient: null,
+			selectedAddress: '',
+			authorized: false
+		}
+	},
+	getters: {
+		client: state => state.activeClient,
+		address: state => state.selectedAddress
+	},
+	mutations: {
+		SET_ACTIVE_WALET(state, wallet) {
```

Comment:
Typo of `SET_ACTIVE_WALLET`

Reply:
good catch...was not actually using that mutation yet (it's part of the localStorage store/restore workflow) hence I never noticed...

Reply:
Fixed in https://github.com/tendermint/vue/pull/49/commits/351a6749e50ac8f610724bd7cae204a2fc55df74